WordPress/Classes/ViewRelated/NUX/SiteCreationCreateSiteViewController.swift

```diff
@@ -1,350 +0,0 @@
-    private let completedLabelTextColor = WPStyleGuide.darkGrey()
-
-    override var sourceTag: WordPressSupportSourceTag {
-        get {
-            return .wpComCreateSiteCreation
-        }
-    }
-
-    // MARK: - View
-
-    override func viewDidLoad() {
-        super.viewDidLoad()
-
-        configureView()
-        setLabelText()
-    }
-
-    override func viewWillAppear(_ animated: Bool) {
-        super.viewWillAppear(animated)
-
-        // If there is a status, retry from the last (i.e. failed) status.
-        if let lastStatus = lastStatus {
-
-            let siteCreationService = SiteCreationService(managedObjectContext: ContextManager.sharedInstance().mainContext)
-
-            switch lastStatus {
-            case .settingTagline:
-                siteCreationService.retryFromTagline()
-            case .settingTheme:
-                siteCreationService.retryFromTheme()
-            case .syncing:
-                siteCreationService.retryFromAccountSync()
-            default:
-                break
-            }
-        } else {
-            // If we have no status, start from the beginning.
-            setDefaultLabelStyle()
-            createSite()
-        }
-    }
-
-    // MARK: - Navigation
-
-    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
-        super.prepare(for: segue, sender: sender)
-
-        if let vc = segue.destination as? SiteCreationEpilogueViewController {
-            vc.siteToShow = newSite
-        }
-
-        if let vc = segue.destination as? NoResultsViewController {
-            let title = NSLocalizedString("Something went wrong…", comment: "Primary message on site creation error page.")
-            let errorButtonTitle = NSLocalizedString("Try again", comment: "Button text on site creation error page.")
-            let imageName = "site-creation-error"
-
-            vc.delegate = self
-            vc.configure(title: title, buttonTitle: errorButtonTitle, subtitle: errorMessage, image: imageName)
-            vc.showDismissButton()
-
-            // Add the WP logo to the navigation bar
-            let image = WordPressAuthenticator.shared.style.navBarImage
-            let imageView = UIImageView(image: image.imageWithTintColor(UIColor.white))
-            vc.navigationItem.titleView = imageView
-        }
-    }
-
-}
-
-// MARK: - View Configuration Extension
-
-private extension SiteCreationCreateSiteViewController {
-
-    func configureView() {
-        WPStyleGuide.configureColors(for: view, andTableView: nil)
-        addWordPressLogoToNavController()
-        // Remove help button.
-        navigationItem.rightBarButtonItems = nil
-        // Remove Back button. There's no going back now!
-        navigationItem.hidesBackButton = true
-    }
-
-    func setLabelText() {
-        layingFoundationLabel.text = NSLocalizedString("Laying site foundation...", comment: "Text shown during the site creation process when it is on the first step.")
```

Comment:
The removed strings should be automatically removed from `Localizable.strings`, right?

Reply:
They should be automatically removed when the strings are updated through our scripts, yes.  It's not a manual process.